src/components/ViewSource/ViewSource.js

```diff
@@ -52,7 +54,10 @@ const ViewSource = ({
   useEffect(() => {
     setIsMarcLoading(true);
 
-    mutator.marcRecord.GET()
+    const tenantId = instance?.tenantId;
+    const { okapi: { tenant, token, locale } } = stripes;
+
+    mutator.marcRecord.GET({ headers: getHeaders(tenantId || tenant, token, locale) })
```

Comment:
`ViewSource` is used in 2 places. This code also changes the tenant for holdings, is it necessary?

Reply:
Hi @Dmytro-Melnyshyn thank you for catching it, updated code